c2corg_ui/templates/base.html

```diff
@@ -109,6 +118,15 @@
     <script src="${request.static_url('%s/angular-messages/angular-messages.min.js' % node_modules_path)}"></script>
     <script src="${request.static_url('%s/angular-cookies/angular-cookies.min.js' % node_modules_path)}"></script>
     <script src="${request.static_url('%s/typeahead.js/dist/typeahead.bundle.min.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/min/moment.min.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/angular-moment/angular-moment.min.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/ca.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/fr.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/en-gb.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/de.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/es.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/eu.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/moment/locale/it.js' % node_modules_path)}"></script>
     <script src="${request.static_url('c2corg_ui:static/build/build.js')}"></script>
```

Comment:
does it happen to every outing or just to a particular one ? i don't have this issue